roles/smallfile-bench/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Generate smallfile test input yaml
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        namespace: '{{ operator_namespace }}'
+        name: smallfile-test
+      data:
+        smallfilejob: "{{ lookup('template', 'smallfilejob.yaml.j2') }}"
+  when: smallfile.clients > 0
+
+- name: Start Run(s)
+  k8s:
+    definition:
+      kind: Job
+      apiVersion: batch/v1
+      metadata:
+        name: '{{ meta.name }}-smallfile-client--benchmark'
+        namespace: '{{ operator_namespace }}'
+      spec:
+        ttlSecondsAfterFinished: 600
+        template:
+          metadata:
+            labels:
+              app: smallfile-benchmark
+          spec:
+            containers:
+            - name: benchmark-server
+              image: "quay.io/nichawla/smallfile_container:testing"
+              command: ["/bin/sh", "-c"]
+              workingDir: /root/smallfile-master/
+              args:
+                 - "mkdir /mnt/test_smallfile/;cat /mnt/smallfile/smallfilejob ; python smallfile_cli.py --yaml-input-file /mnt/smallfile/smallfilejob --output-json /root/SMALLFILE_RUN_RESULT; ls -R /mnt/"
```

Comment:
this command is missing ```--host-set pods.list --launch-by-daemon Y``` .    To see what I mean, see examples/docker/run-smallfile-client-tests.sh in the source tree.   It demonstrates all this.  For example, 

/run-smallfile-client-tests.sh /var/tmp/smf ~/smallfile quay.io/bengland2/smallfile:20190411.b